debezium-microbenchmark-oracle/src/main/java/io/debezium/performance/connector/oracle/parser/LogMinerDmlParserPerf.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.performance.connector.oracle.parser;
+
+import java.util.concurrent.TimeUnit;
+
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
+
+import io.debezium.connector.oracle.logminer.parser.DmlParser;
+import io.debezium.connector.oracle.logminer.parser.LogMinerDmlParser;
+
+/**
+ * A basic test to determine the performance of the new LogMiner DML parser for Oracle.
+ *
+ * @author Chris Cranford
+ */
+public class LogMinerDmlParserPerf {
+
+    @State(Scope.Thread)
+    public static class ParserState {
+        public DmlParser dmlParser;
+        public String insertDml;
+        public String updateDml;
+        public String deleteDml;
+        public String txId;
+
+        @Param({ "1", "2", "5", "10", "20", "50" })
+        public int columnCount;
+
+        @Setup(Level.Trial)
+        public void doSetup() {
+            dmlParser = new LogMinerDmlParser();
+            this.insertDml = insertStatement();
+            this.updateDml = updateStatement();
+            this.deleteDml = deleteStatement();
+            this.txId = "1234567890";
+        }
+
+        private String insertStatement() {
+            final StringBuilder sb = new StringBuilder("insert into \"DEBEZIUM\".\"TEST\"(\"ID\"");
+            for (int i = 0; i < columnCount; ++i) {
+                sb.append(",").append("\"C").append(i).append("\"");
+            }
+            sb.append(") values (");
+            sb.append("'").append(1).append("'");
+            for (int i = 0; i < columnCount; ++i) {
+                sb.append(",").append("'V").append(i).append("'");
```

Comment:
Values probably should be a bit longer to be closer to what you'd actually have. Like some strings with 50 char or so.